ts/src/connections/tunnelConnectionSession.ts

```diff
@@ -298,4 +323,25 @@ export class TunnelConnectionSession extends TunnelConnectionBase implements Tun
             return this.accessToken;
         }
     }
+
+    /** @internal */
+    public createRequestMessageAsync(port: number): Promise<PortForwardRequestMessage> {
```

Comment:
Will this internal status be reflected in d.ts? If we wanted to hide these from the public surface area, we could create a separate internal class that would implement these methods.

Reply:
The `@internal` tag causes them to be omitted from the `.d.ts`.

It could be possible to refactor to a helper class, but `@internal` works fine.